docs/ELRS_RECEIVERS.md

```diff
@@ -0,0 +1,63 @@
+# mLRS Documentation: ELRS Receivers #
+
+([back to main page](../README.md))
+
+## Supported ELRS Receivers ##
+
+As of April 2024, ELRS receivers with an ESP8285 MCU are experimentally supported by mLRS. The following targets are supported:
+
+- Generic 900
+- Generic 900 PA
+- Generic 2400
+- Generic 2400 PA
+
+To determine if your receiver hardware is supported, go to [ELRS Targets](https://github.com/ExpressLRS/targets/blob/master/targets.json) and find the layout file of your hardware.
+
+Note: ELRS receivers that use an ESP32 MCU are under active development.
+
+## Recommended ELRS Receivers ##
+
+The following receivers are recommended as they support >= 100 mW transmit power and have been validated by the mLRS team.
+
+<table>
+  <tbody>
+    <tr>
+      <td>Product Name</td>
+      <td>Frequency Band</td>
+      <td>Transmit Power</td>
+    </tr>
+    <tr>
+      <td>BayckRC 915M Nano Pro</td>
+      <td>868/915 MHz</td>
+      <td>27 dBm (500 mw)</td>
+    </tr>
+    <tr>
+      <td>SpeedyBee Nano</td>
+      <td>2.4 GHz</td>
+      <td>20 dBm (100 mw)</td>
+    </tr>
+  </tbody>
+</table>
```

Comment:
would it maybe be appropriate to incluude a link to the original vendors page? With these thousands of devices I find it difficult to differentiate genuine ones from clones. (just googling the name produces 1000 entries with many being dubios) 
just a thought, might be also seen inapproriate

Reply:
I wouldn't mind but the challenge here is that Bayck doesn't seem to have a website or even an official Ali store.

Reply:
ah ... oh ... hm ... well, that's why it was just a thought LOL

Reply:
Maybe should have gone with iflight 500mw rx ;-)